platform/posix/clock_posix.c

```diff
@@ -34,6 +34,13 @@
 /* Platform clock include. */
 #include "clock.h"
 
+/*
+ * Time conversion constants.
+ */
+#define NANOSECONDS_PER_SECOND         ( 1000000000 ) /**< @brief Nanoseconds per second. */
```

Comment:
Is this macro required?

Reply:
No, it's not used anywhere here so I will remove it